ksm-throttler/_service-template

```diff
@@ -0,0 +1,24 @@
+<!--- XML Structure defined here: https://en.opensuse.org/openSUSE:Build_Service_Concept_SourceService --> 
+<services>
+ <service name="tar_scm">
+    <param name="scm">git</param>
+    <param name="url">https://github.com/kata-containers/ksm-throttler.git</param>
+    <param name="filename">kata-ksm-throttler</param>
+    <param name="versionformat">@VERSION@.git+%h</param>
+    <param name="revision">@REVISION@</param>
+ </service>
+ <service name="recompress">
+    <param name="file">*.tar*</param>
+    <param name="compression">gz</param>
+  </service>
+ <service name="download_url">
+    <param name="protocol">https</param>
+    <param name="host">storage.googleapis.com</param>
+    <param name="path">golang/go@GO_VERSION@.linux-amd64.tar.gz</param>
+ </service>
+ <service name="verify_file">
+    <param name="file">_service:download_url:go@GO_VERSION@.linux-amd64.tar.gz</param>
```

Comment:
And again here.

Reply:
This are a diffcult according to @erick0z , this is a URL that will download OBS, being a variable or not will be possible to just upload one go, OBS will try to use the go we provide in the tarball but depedning in the build target, it will be a x86_64 VM or other architecture. We may need to rework this to add an URL for each distro we support?